addons/account/models/account_move_line.py

```diff
@@ -78,6 +78,7 @@ class AccountMoveLine(models.Model):
         compute='_compute_account_id', store=True, readonly=False, precompute=True,
         inverse='_inverse_account_id',
         index=True,
+        auto_join=True,
```

Comment:
so no security issue then?

Reply:
I think that it is fine. If you have access to `account.move.line`, you should have access to those auto joined fields